api/v1/payments.md

```diff
@@ -47,6 +47,7 @@
   },
   "form_id": 112,
   "custom_id": "GHS430",
+  // Checkout data is available only if the payment was from a one-time checkout
```

Comment:
Should we say that there will be a customer available if it is a recurring payment and that customer will have the checkout data?

Reply:
Sure. Will do.